### ISSUE TITLE
TypeError: Cannot read property 'on' of undefined

### ISSUE THREAD

**Issue by Turbo87** [ember-cli/ember-exam#378]
I ran `ember install ember-exam` on a relatively simple Ember addon project, then tried out `ember exam` and this seems to crash when trying to run the tests:

```
Randomizing tests with seed: false
Environment: test
cleaning up...
/Users/tbieniek/Code/qunit-console-grouper/node_modules/ember-exam/lib/commands/exam.js:358
        this.process.on('processExit', browserFailureHandler.bind(this));
                     ^

TypeError: Cannot read property 'on' of undefined
    at BrowserTestRunner.events.tests-start (/Users/tbieniek/Code/qunit-console-grouper/node_modules/ember-exam/lib/commands/exam.js:358:22)
    at Socket.emit (events.js:203:15)
    at /Users/tbieniek/Code/qunit-console-grouper/node_modules/socket.io/lib/socket.js:528:12
    at process._tickCallback (internal/process/next_tick.js:61:11)
```

**Comment by Turbo87** [comment]
It seems like the issue was `import start from 'ember-exam/test-support/start';` missing from the `test-helper.js`, but the error message isn't exactly straight forward and could probably mention that this is what might be missing.

**Comment by MichalBryxi** [comment]
My `test-helper.js` looks like:

```js
import Application from '../app';
import config from 'dummy/config/environment';
import start from 'ember-exam/test-support/start';
import { setApplication } from '@ember/test-helpers';

setApplication(Application.create(config.APP));

start();
```

And I'm getting: 

```
node_modules/ember-exam/lib/commands/exam.js:358
        this.process.on('processExit', browserFailureHandler.bind(this));
                     ^

TypeError: Cannot read property 'on' of undefined
    at BrowserTestRunner.events.tests-start (/node_modules/ember-exam/lib/commands/exam.js:358:22)
    at Socket.emit (events.js:194:15)
    at Socket.EventEmitter.emit (domain.js:441:20)
    at node_modules/socket.io/lib/socket.js:528:12
    at process._tickCallback (internal/process/next_tick.js:61:11)
```

**Comment by step2yeung** [comment]
Following up on this:

I believe @choheekim provided a fix for this issue with #377 which landed on v4.0.4. 
@MichalBryxi @Turbo87 would you be able to try out v4.0.4 and let us know if this is still an issue?

**Comment by choheekim** [comment]
@MichalBryxi Do you mind sharing what ember-exam version you are using?

**Comment by MichalBryxi** [comment]
Certainly @step2yeung, it is: `"ember-exam": "4.0.3",`

**Comment by choheekim** [comment]
@MichalBryxi ember-exam@4.0.3 has an issue, do you mind trying v4.0.4 and let me know if you are seeing the same issue?

**Comment by jherdman** [comment]
I'm still seeing this error with a combo of ember-exam 4.0.4 and ember-cli 3.13.

**Comment by step2yeung** [comment]
@jherdman mind sharing the stacktrace for your error?

**Comment by jherdman** [comment]
<img width="960" alt="Screen Shot 2019-10-08 at 8 57 50 PM" src="https://user-images.githubusercontent.com/3300/66443780-5a7b9100-ea0e-11e9-9f6e-e2225cc2fff6.png">

Apologies for the screen shot. It's kinda mangled, but that's the extent of it.

**Comment by choheekim** [comment]
@jherdman Thank you for the screen shot! just a qq
1. What command are you running to execute tests?
2. How do you launch a browser? Is the testem brining the browser up?

**Comment by jherdman** [comment]
1. `ember exam --server`
2. Manually starting the browser, and then typing/pasting the URL

**Comment by choheekim** [comment]
@jherdman so the problem is this.process is not available when a browser is launched manually. I will put a pr up to validate whether `this.process` is available when registering events for `processExit` and `processError`.

**Comment by choheekim** [comment]
fyi, a pr is being worked on and reviewed
https://github.com/ember-cli/ember-exam/pull/388

**Comment by sukima** [comment]
I am able to reproduce on 4.0.5 with `ember exam --server --launch=false`.

**Comment by choheekim** [comment]
@sukima Could you try with v4.0.5? the fix was released on v4.0.5

**Comment by jherdman** [comment]
I can confirm on 4.0.5

**Comment by choheekim** [comment]
I was able to reproduce the error. I will put a pr for a fix and have it release asap

**Comment by jherdman** [comment]
Any word on a fix?

**Comment by step2yeung** [comment]
I believe the fix is here #409
